### DISCUSSION TITLE
Get generated project's destination within copier.yml

### DISCUSSION THREAD

**Discussion by jboys** [copier-org/copier#478] [Q&A]
How do I get the generated project's destination from within `copier.yml`?

For example, when running `copier path/to/project/template path/to/destination` I want to use `destination` as the default answer to my `project_name` question.

I want to do something like this:

```
# copier.yml

project_name: [[ _dst_path | basename ]]
```

**Comment by yajo** [comment]
Have you tried `_copier_conf.dst_path`?

**Reply by jboys** [reply]
Yes, but I get the error `'_copier_conf' is undefined`

**Reply by yajo** [reply]
Probably it's not available while in `copier.yaml`, which has some special constraints regarding to the environment it runs into.

**Reply by pkubat** [reply]
@jboys  Did you ever resolve this?  I was looking to enforce the destination naming format was enforce and I have the same issue with `_copier_conf`.  Thanks

**Reply by yajo** [reply]
For that use case, maybe tasks will help.
